tlvc/fs/buffer.c

```diff
@@ -438,23 +442,25 @@ struct buffer_head *get_free_buffer(void)
 	     * Skip if we're running with L1 buffers only.
 	     * (in which case brelseL1_index() is a dummy anyway)
 	     */ 
-		printk("RelL1:");
+		//printk("RelL1:");
 		for (i=0; i<nr_mapbufs; i++)
 			brelseL1_index(i, 1);
-		printk("\n");
+		//printk("\n");
 		break;
 #else
 		continue;
```

Comment:
Just happened to notice this - in the case of L1 buffers only, `continue` would continue the while loop but without incrementing sync_loop. It would seem that would cause an infinite loop if ebh->b_next_lru were NULL (meaning no buffers available)? Perhaps should be `break` like the other switch cases?

Reply:
Indeed, @ghaerr - much appreciated. A reminder that the L1 only setting is 'under-tested'. Thank for reviewing!